client/components/tinymce/plugins/mentions/plugin.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import tinymce from 'tinymce/tinymce';
+import ReactDom from 'react-dom';
+import React from 'react';
+import { Provider as ReduxProvider } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Mentions from './mentions';
```

Comment:
Error above is gone, but we need to handle when suggestions is a falsey value instead of an array:

![screen shot 2017-02-28 at 3 12 02 pm](https://cloud.githubusercontent.com/assets/1270189/23438796/5e3f71a6-fdc8-11e6-908c-666f8e3c7e41.png)